tests/cli_test.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from tests.draw_test import TestBase
+import unittest
+from worldengine import __main__
+from worldengine.cli.main import main
+
+
+class TestCLI(TestBase):
+
+    def setUp(self):
+        super(TestCLI, self).setUp()
+        self.world = "%s/seed_28070.world" % self.tests_data_dir
+
+    def test__main__(self):
```

Comment:
nitpick/pointless: I am not sure if we should include the underscores in the name. I am afraid someone could think this method is private because it terminates with underscores.